src/components/DateInput.jsx

```diff
@@ -88,15 +90,17 @@ export default class DateInput extends React.Component {
     const {
       id,
       placeholder,
-      dateValue,
+      displayValue,
+      inputValue,
       focused,
       showCaret,
       onFocus,
       disabled,
       required,
     } = this.props;
 
-    const value = dateValue || dateString;
+    const displayText = displayValue || inputValue || dateString || placeholder;
```

Comment:
does this also need an `|| ''` at the end?

yo dawg, i heard you like defaults